tests/kafkatest/services/kafka/kafka.py

```diff
@@ -102,6 +104,7 @@ def __init__(self, context, num_nodes, zk, security_protocol=SecurityConfig.PLAI
             'SASL_SSL': Port('SASL_SSL', 9095, False)
         }
 
+        self.path = create_path_resolver(self.context)
```

Comment:
Similar to the mixin note above, it'd be nice if we didn't have to import this and create path resolvers all over the place -- there's a lot of redundancy that I think we could remove by having a mixin or common base class.